src/oversampling_methods/random_walk/interface_mlj.jl

```diff
@@ -46,11 +46,12 @@ MMI.metadata_model(
     RandomWalkOversampler,
     input_scitype = Union{
         Table(Union{Infinite, Finite}),
+        AbstractVector
     },
     output_scitype = Union{
         Table(Union{Infinite, Finite}),
+        AbstractVector
     },
-    target_scitype = AbstractVector,
     load_path = "Imbalance.MLJ.RandomWalkOversampler"
 )
```

Comment:
Why is this one not repaired in the same pattern as the others? Don't you need `Tuple` somewhere?

Reply:
Oops! I did this shortly before sleeping and looks like I took Union{ to be Tuple{ as it was indented similar to other methods.